docs/css/site.css

```diff
@@ -1,16 +1,11 @@
-.sidebar-content {
-  height: 100%;
-  max-height: unset;
-}
```

Comment:
This can be added back.  However, would it be better to include this in core pelican css? We could possibly check for the presence of the footer and adjust as needed?  Maybe something like `.sidebar-content + .sidebar_footer {}`